invenio_oauthclient/contrib/github.py

```diff
@@ -146,3 +156,24 @@ def authorized(resp, remote):
             )
 
     return authorized_signup_handler(resp, remote)
+
+
+def disconnect_handler(remote, *args, **kwargs):
+    """Handle unlinking of remote account."""
+    if not current_user.is_authenticated:
+        return current_app.login_manager.unauthorized()
+
+    remote_account = RemoteAccount.get(user_id=current_user.get_id(),
+                                       client_id=remote.consumer_key)
+    external_method = 'github'
+    external_ids = [i.id for i in current_user.external_identifiers
+                    if i.method == external_method]
+
+    if external_ids:
+        oauth_unlink_external_id(dict(id=external_ids[0],
+                                      method=external_method))
+    if remote_account:
+        with db.session.begin_nested():
+            remote_account.delete()
+
```

Comment:
I added the commit here on oauthclient `disconnect()` (because also orcid needed commit): https://github.com/inveniosoftware/invenio-oauthclient/pull/54/commits/7c22ee2f526b3a3360ba6edbe42c16f221fea49e#diff-f2d8bbac55a20ee4eb23a17827b21543R151
WDYT?